.travis.yml

```diff
@@ -2,11 +2,10 @@ language: node_js
 
 matrix:
   include:
-    - node_js: "4"
-    - node_js: "5"
     - node_js: "6"
     - node_js: "7"
```

Comment:
Now that 8 is LTS, I would remove 7, and only keep 9 for future durections

Reply:
I'm not sure about this. I know of people inside companies using OC (cough cough) using node 7 in production 🤣

Reply:
Lets do this later